social-circus/src/redux/actions/authActions.jsx

```diff
@@ -39,8 +54,9 @@ export const registerUser = async (userData, navigate, dispatch, login) => {
 const createProfile = async (user, uid) => {
   try {
     await setDoc(doc(db, "users", uid), {
-      firstName: user.firstName,
-      lastName: user.lastName,
+      firstname: user.firstName,
+      lastname: user.lastName,
+      displayName: user.firstName + " " + user.lastName,
```

Comment:
instead of storing a extra variable in the backend, you can get firstname and lastname in the client side and show it in the client side

Reply:
Yes, that makes more sense . Thank You.